xc/xc7/tests/soc/litex/mini_ddr/minilitex_ddr_arty.xdc

```diff
@@ -1,218 +1,225 @@
-# ## serial:0.tx
-#set_property LOC D10 [get_ports serial_tx]
+set_property LOC H5 [get_ports {led[0]}]
```

Comment:
I believe you should eliminate the use of the pcf file for this example to show that the flow no longer requires PCF files?